### PR TITLE
Changing the --with-stdout behaviour so it works with non-init commands

### PR DESCRIPTION
**Pull request by nickdgriffin**
Whilst testing out my compose v2 PR I noticed that commands like "./adop compose down" would fail because the list of compose file to use weren't being set (the variables being local to the init function), so it was defaulting and missing off the volume information (which in v2 is in both the main file, to declare it, and the override file, to define it).

This PR is a fix for this problem by allowing the -l flag to "./adop compose" to be set to "stdout" which will disable the logging driver override file entirely, and also to make the "./adop compose init --with-stdout" command align to this functionality. This means that "./adop compose init" followed by "./adop compose down" in Compose v2 will not error.

If #166 is merged first this PR will need updating, and vice versa.

### CODE REVIEW

**Comment by nickdgriffin** [comment]
Not expected, and admittedly not tested without -d, so I'll give it another pass and see what I missed/broke.

Thanks!

**Comment by nickdgriffin** [comment]
I've figured it out! So, what is happening is I've actually made that function work the way it is meant to which means that when doing an "up" it's just doing that against the ELK compose file and that's all it'll run. 

This of course means that you don't actually have all of the services running, and you can't access them either due to a lack of Nginx, so I'll put in some logic that will make it run ELK with "up -d" instead so that the main stack is up'd.

Basically being bitten by the double "docker-compose" here: https://github.com/Accenture/adop-docker-compose/blob/master/cmd/compose#L290

**Comment by nickdgriffin** [comment]
Added a fix for the issue with "up", I'll squash it down assuming it works.

Basically if the requested command for run_compose is "up" it'll check if the array contains "-d" and if it doesn't add it on just for ELK - that way we always launch ELK independently before proceeding with the main Compose file. There's not much point allowing a regular "up" on the ELK stack because it won't be reachable without Nginx anyway.

**Comment by nickdgriffin** [comment]
I think this is ready for another review. Open to suggestion/opinion on whether it should be squashed or not as the two commits can stand on their own and could have potentially been separate PRs.

**Comment by subodh-hatkar** [comment]
Both of the following commands work as expected, no issues found.

```
/adop compose -m ADOP-PR-181 init --with-stdout

./adop compose -m ADOP-PR-181 -l stdout up
```

@nickdgriffin : Please rebase your branch as it is out of date.

Will test it again after rebase.

**Comment by nickdgriffin** [comment]
@subodh-hatkar Done!
